### PR TITLE
support yaru for xfce

### PR DESCRIPTION
**Pull request by Muqtxdir**
Hi, 

This PR adds all the assets required for window-decorations on xfce
*Say hello to Yaru on XFCE*
### TO-DO

- [x] default assets for **xfwm4**
- [x] dark assets for **xfwm4**
- [x] add xfwm4 to `meson.build`
- [x] update `/debian`

#### Yaru:
Screenshot from 2021-07-30 16-39-33

#### Yaru-dark:
Screenshot from 2021-07-30 16-39-00

Closes: #2770

### CODE REVIEW

**Comment by Muqtxdir** [comment]
made the `xfwm4` assets in the size of _**46x40**_ to match the headerbar-size of **GTK-3** theme and converted the SVGs into PNGs to reduce size and enabled xfwm4 option in meson
xfwm4-assets.

It's completed now so anyone is welcome to try-out this branch. (*I tested this branch on Xubuntu 21.04*)
#### To change the theme in xfce-desktop:
- Open Appearance app to change GTK-3 theme to **Yaru** or **Yaru-dark**
- Open Window-manager app and change theme to **Yaru** or **Yaru-dark**

**Comment by Jupi007** [comment]
Awesome work @Muqtxdir 😃

**Comment by ItzSwirlz** [comment]
@Muqtxdir By the way, there is CSS triggers for XFCE: https://github.com/UbuntuBudgie/pocillo-gtk-theme/blob/master/src/gtk-3.0/sass/apps/_xfce.scss

**Comment by Muqtxdir** [comment]
Thanks @ItzSwirlz I'll look into those files tomorrow 👍

**Comment by ItzSwirlz** [comment]
> Thanks @ItzSwirlz I'll look into those files tomorrow 👍

Awesome, I just recommend doing Thunar in a separate file in case anything CSS-wise changes upstream :)

**Comment by Muqtxdir** [comment]
in `_xfce.scss`, I made the top-bar color similiar to the one in Gnome Top-bar (*before it was white, see screenshot in initial-post*) and chose to not overwrite ***Thunar's*** styling as it looks good without any tweaks.

#### screenshot: Yaru
Screenshot_2021-08-01_19-18-22

#### screenshot: Yaru-dark
Screenshot_2021-08-01_19-18-48

**Comment by Initu-Castilhos** [comment]
YARU-MATE would look good with a dark panel too.

**Comment by Muqtxdir** [comment]
> YARU-MATE would look good with a dark panel too.

@Initu-Castilhos could you please open an issue for such discussion for MATE dark top-panel. So that others can notice and share their opinions and thoughts on your suggestion.

**Comment by Muqtxdir** [comment]
should I merge this PR?

**Comment by Jupi007** [comment]
I'm really sorry @Muqtxdir. I totally forgotten to review your PR 😔
Will take a look tonight, I promise :)

File: meson_options.txt [hunk 1]
```diff
@@ -11,3 +11,4 @@ option('communitheme_compat', type: 'boolean', value: false, description:'build
 option('default', type: 'boolean', value: true, description:'build Yaru gtk default flavour')
 option('dark', type: 'boolean', value: true, description:'build Yaru gtk dark flavour')
 option('ubuntu-unity', type: 'boolean', value: false, description:'build Yaru with Unity assets')
+option('xfwm4', type: 'boolean', value: true, description:'build Yaru with xfwm4 assets')
```
Comment (Jupi007) [inline meson_options.txt:14]: 🤔 Maybe make it disabled by default?

**Review by Jupi007 — COMMENTED** [review]
Maybe make those headerbar less big:

Capture d’écran du 2021-08-18 18-45-03

35px height could be better imo.

Except this little detail, everything looks perfect!
That's an awesome work @Muqtxdir 💪

File: meson_options.txt (hunk 1 above)
Comment (Muqtxdir) [reply meson_options.txt:14]: sure thing 👍

**Comment by Muqtxdir** [comment]
> Maybe make those headerbar less big:
> 
> Capture d’écran du 2021-08-18 18-45-03
> 
> 35px height could be better imo.
> 
> Except this little detail, everything looks perfect!
> That's an awesome work @Muqtxdir 💪

I'll reduce those to 35px tomorrow

**Comment by Muqtxdir** [comment]
done!

**Comment by Jupi007** [comment]
Thanks a lot for your work @Muqtxdir 
I just tested that and the size is really much better imo.

But window buttons are now blurred:

Capture d’écran du 2021-09-07 09-14-50

It is maybe related to the odd size. Can you try 36px height (😥 I'm really sorry, I hope it's not too long for you to resize all these assets, if you want I can do it).

And if I can make a last suggestion, maybe reduce window controls padding like this:

Capture d’écran du 2021-09-07 09-13-03

And it will be perfect after :)

**Comment by Muqtxdir** [comment]
no problem @Jupi007 , I'll try at 36px 👍

**Comment by Jupi007** [comment]
Fixed :)

Capture d’écran du 2021-09-07 16-46-24

**Review by Jupi007 — APPROVED** [review]

**Comment by Jupi007** [comment]
@Muqtxdir Fell free to check my work just in case :)

**Comment by Muqtxdir** [comment]
@Jupi007, Thanks for helping today! It looks really sharp but I noticed `hide-inactive` in **dark** and **default** is ***35px***. could you fix that ?

**Comment by Jupi007** [comment]
Fixed :)

**Comment by Muqtxdir** [comment]
Thanks @Jupi007, LGTM :+1:

**Comment by Jupi007** [comment]
And Yaru now officially support Xfce :tada:

**Comment by Kuna42** [comment]
(Maybe it is not the right place, but i have no idea, where i can ask this.)

I like Yaru-dark on xfce, except the taskbar.
I like the [old taskbar](https://user-images.githubusercontent.com/54065734/127644825-6098f137-7fa4-4090-9a76-417ddfbaed36.png) from the first post.
And i dislike the [current version](https://user-images.githubusercontent.com/54065734/127773375-539018aa-0eb8-404f-b66e-b4d2814a17ee.png) of the highlighted program in the dark theme.
How can i download the theme with that different task bar, or could you say me how i can change this for myself?

Please don't understand me wrong, i thank you, that you develop Yaru for xfce. And if i have the knowledge to change it by myself, i would do it. But i have not the knowledge. :/

**Pull request merged**